src/components/tx/DecodedTx/index.tsx

```diff
@@ -41,9 +41,9 @@ const DecodedTx = ({
   showMultisend = true,
   showMethodCall = false,
 }: DecodedTxProps): ReactElement => {
-  const isMultisend = !!decodedData?.parameters?.[0]?.valueDecoded
   const chainId = useChainId()
-  const isMethodCallInAdvanced = !showMethodCall || isMultisend
+  const isMultisend = !!decodedData?.parameters?.[0]?.valueDecoded
+  const isMethodCallInAdvanced = !showMethodCall || (isMultisend && showMultisend)
```

Comment:
The showMultisend here is confusing. Now the name is `isMethodCallInAdvanced`, but since showMultisend could be false this would also be false, despite the method call being in advanced. I'm more for havign showMultisend inside of the render.

Reply:
Just to make sure we understand the same logic. We move the method name and parameters to Advanced details in two cases:
* When `showMethodCall` is set to false in props
* When it's a multi-send and `showMultisend` from props is true

How would you make this less confusing?

Reply:
Let's leave it like this for now as @clovisdasilvaneto and I are refactoring this more comprehensively.